README.md

```diff
@@ -217,6 +218,10 @@ See the [apiclient documentation](sources/api/apiclient/) for more details.
 
 If you are running the Kubernetes variant of Bottlerocket, you can use the [Bottlerocket update operator](https://github.com/bottlerocket-os/bottlerocket-update-operator) to automate Bottlerocket updates.
 
+#### Bottlerocket ECS Updater
```

Comment:
I think we should just get rid of this and the brupop note just above; it's weird to have it duplicated above and below the update API subsection.  I'd also recommend a header after "There are several ways..." and before the updater notes, perhaps `#### Automated Updates`.

Reply:
Done